jwst/assign_wcs/nirspec.py

```diff
@@ -381,14 +381,15 @@ def get_open_fixed_slits(input_model, slit_y_range=[-.55, .55]):
     """ Return the opened fixed slits."""
     if input_model.meta.subarray.name is None:
         raise ValueError("Input file is missing SUBARRAY value/keyword.")
+
     slits = []
     ylow, yhigh = slit_y_range
 
-    s2a1 = Slit('S200A1', 0, 0, 0, 0, ylow, yhigh, 5)
-    s2a2 = Slit('S200A2', 1, 0, 0, 0, ylow, yhigh, 5)
-    s4a1 = Slit('S400A1', 2, 0, 0, 0, ylow, yhigh, 5)
-    s16a1 = Slit('S1600A1', 3, 0, 0, 0, ylow, yhigh, 5)
-    s2b1 = Slit('S200B1', 4, 0, 0, 0, ylow, yhigh, 5)
+    s2a1 = Slit('S200A1', 0, 0, 0, 0, ylow, yhigh, 5, 1)
+    s2a2 = Slit('S200A2', 1, 0, 0, 0, ylow, yhigh, 5, 2)
+    s4a1 = Slit('S400A1', 2, 0, 0, 0, ylow, yhigh, 5, 3)
+    s16a1 = Slit('S1600A1', 3, 0, 0, 0, ylow, yhigh, 5, 4)
+    s2b1 = Slit('S200B1', 4, 0, 0, 0, ylow, yhigh, 5, 5)
```

Comment:
What is the extra argument that's been added at the end of each of these calls? Are you just using the integer values 1-5 as the source_id? As opposed to using the string versions (S200A1, etc.) of the slit names?

Reply:
Yes, I am assigning integer values as source_id. Just to be consistent with MOS data.